include/pybind11/attr.h

```diff
@@ -272,6 +290,9 @@ struct type_record {
     /// Custom metaclass (optional)
     handle metaclass;
 
+    /// Custom type setup.
+    ::pybind11::custom_type_setup::callback custom_type_setup;
```

Comment:
Using the same name for two different things is easily confusing. How about:
```
custom_type_setup::callback custom_type_setup_cb;
```
Or `custom_type_setup_callback` fully spelled out (I actually like that best; no need to go to a shortcut for something that's reference only a couple times).

Reply:
Done.